server/routes/index.js

```diff
@@ -1,18 +1,20 @@
 import express from 'express';
 import swaggerUI from 'swagger-ui-express';
 import yaml from 'yamljs';
-import userRouter from "./userRoute"
+import userRouter from './userRoute';
+import socialRouter from './socialRoute';
 
```

Comment:
I was thinking you were going to add all the routes in one file.
Nice work though

Reply:
The route doesn't start with auth
It starts with oauth
That's why I separated them